Extension/src/packageManager.ts

```diff
@@ -376,7 +370,7 @@ export class PackageManager {
 
                 zipfile.readEntry();
 
-                zipfile.on('entry', (entry: yauzl.Entry) => {
+                zipfile.on('entry', async (entry: yauzl.Entry) => {
```

Comment:
Does zipfile.on calls this lambda as async?  If not, the lambda may not be awaited, so would return to the caller in an incomplete state.  (Not sure if that is an issue in this particular case)

Reply:
open a new bug 
leave it
revert it